### ISSUE TITLE
The problem with the picture in the Roadsters game.

### ISSUE THREAD

**Issue by SatanNintendo** [ata4/angrylion-rdp-plus#149]
Hello! When I launch the Roadsters game, the game shows some dots in the menu, but when it comes to the game itself, then the picture becomes illegible there, in columns.
I tried to run on PJ64 and on RMG, on Windows 7 x64 and on Windows 11, the result was the same. The plugin uses the latest build - angrylion-plus_v1.6-25.
[https://hostingkartinok.com/show-image.php?id=ce7832d9d17ec8cc91687cc6e7e7926e](url)

**Comment by Jj0YzL5nvJ** [comment]
It doesn't work in **VI Mode** = `Filtered`... works in `Unfiltered`

<details>
<summary>VI Mode...</summary>

* Filtered
<img src="https://i.imgur.com/mRx324Z.png" alt="" class="">

* Unfiltered
<img src="https://i.imgur.com/vI9gSDu.png" alt="" class="">

* Filtered
<img src="https://i.imgur.com/eku8xwC.png" alt="" class="">

* Unfiltered
<img src="https://i.imgur.com/xa07Wta.png" alt="" class="">

</details>

**Comment by SatanNintendo** [comment]
> It doesn't work in **VI Mode** = `Filtered`... works in `Unfiltered`
> VI Mode...
> 
>     * Filtered
> 
> 
> ![](https://camo.githubusercontent.com/6094c3f6b58c7576c7322c846c34e8462489942281738935b20cbccc0004a7db/68747470733a2f2f692e696d6775722e636f6d2f6d52783332345a2e706e67)
> 
>     * Unfiltered
> 
> 
> ![](https://camo.githubusercontent.com/965e95d889dce91a483c13bc4093e57c5f69eeca55f9b76eb797015baa5cd274/68747470733a2f2f692e696d6775722e636f6d2f764939675344752e706e67)
> 
>     * Filtered
> 
> 
> ![](https://camo.githubusercontent.com/6555431ed69d47d4bcfea6916ca44b2cd58664e814093a82257648b67f02f464/68747470733a2f2f692e696d6775722e636f6d2f656b75387877432e706e67)
> 
>     * Unfiltered
> 
> 
> ![](https://camo.githubusercontent.com/7c2c17a182bc28a651c841541fb104c2bee7d374242deda967ee01d613dd828d/68747470733a2f2f692e696d6775722e636f6d2f786130375774612e706e67)

Yes, indeed, it works fine without a filter. Thanks. But it's still strange, other games mostly work fine with the filter.
Thank you in any case, while i can play without a filter, someday it may also be possible to fix it.

**Comment by Jj0YzL5nvJ** [comment]
The filtered mode is definitely doing something very wrong.

**Comment by SatanNintendo** [comment]
Probably, this can be closed, since today everything works with the filter as it should work.
